modules/apps/rate-limiting/keeper/flow.go

```diff
@@ -0,0 +1,77 @@
+package keeper
+
+import (
+	errorsmod "cosmossdk.io/errors"
+	sdkmath "cosmossdk.io/math"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/cosmos/ibc-go/v10/modules/apps/rate-limiting/types"
+)
+
+// The total value on a given path (aka, the denominator in the percentage calculation)
+// is the total supply of the given denom
+func (k Keeper) GetChannelValue(ctx sdk.Context, denom string) sdkmath.Int {
+	return k.bankKeeper.GetSupply(ctx, denom).Amount
+}
+
+// TODO: Refactor.
```

Comment:
?

Reply:
It has not been planned that this PR requires follow-up refactoring or unit test work, so make sure we are done in this PR with stuff like this :)
i.e. no TODOs plz